src/components/survey/generic/question-set.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from "prop-types";
+import {useEffect, useState} from "react";
+import Responses from "./responses";
+import style from "./style.scss";
+
+export default function QuestionSet({next, questionSet, updateAnswer}) {
+  const questionSetClass = [style.questionSet].join(" ");
+  const [selectedOptions, setSelectedOptions] = useState([]);
+  const setFinished = questionSet.questions.length === selectedOptions.length;
+  const selectOption = (questionId, optionId) => {
+    if(!selectedOptions.includes(questionId)) setSelectedOptions([...selectedOptions, questionId]);
+    updateAnswer(questionId, optionId);
+  };
+
+  useEffect(() => {
+    if(!setFinished) return;
+    next();
+  }, [setFinished]);
+
+  return (
+    <div className={questionSetClass}>
+      <img src={questionSet.setImage} alt={questionSet.text} />
+      <hr />
```

Comment:
Avoid using `hr`'s since the site it's rendered on may style them differently. Instead use a class that gives a border